Migration script/openIMIS migration v1.4.2 - 1.5.0.sql

```diff
@@ -1291,3 +1291,12 @@ BEGIN
 	RETURN 0
 END
 GO
+
+
+--OP-190: BEPHA Policies App: Marital Status shows "-- Select Status--"
```

Comment:
Should you update also uspConsumeEnrollments in this migration?

Reply:
Updated